shared/api/models/__tests__/DateModel.spec.ts

```diff
@@ -411,28 +420,28 @@ describe('DateModel', () => {
         recurrenceRule,
       })
 
-      expect(date.recurrences(3)).toEqual([
-        new DateModel({
+      expect(toUTCSpans(date.recurrences(3))).toEqual([
+        {
           allDay: false,
           recurrenceRule,
-          startDate: DateTime.fromISO('2024-08-12T10:30:00.000+02:00'),
-          endDate: DateTime.fromISO('2024-08-12T12:00:00.000+02:00'),
-          offset: 120,
-        }),
-        new DateModel({
+          startDate: '2024-08-12T10:30:00.000Z',
+          endDate: '2024-08-12T12:00:00.000Z',
+          offset: 0,
+        },
+        {
           allDay: false,
           recurrenceRule,
-          startDate: DateTime.fromISO('2024-09-09T10:30:00.000+02:00'),
-          endDate: DateTime.fromISO('2024-09-09T12:00:00.000+02:00'),
-          offset: 120,
-        }),
-        new DateModel({
+          startDate: '2024-09-09T10:30:00.000Z',
+          endDate: '2024-09-09T12:00:00.000Z',
+          offset: 0,
+        },
+        {
           allDay: false,
           recurrenceRule,
-          startDate: DateTime.fromISO('2024-10-14T10:30:00.000+02:00'),
-          endDate: DateTime.fromISO('2024-10-14T12:00:00.000+02:00'),
-          offset: 120,
-        }),
+          startDate: '2024-10-14T10:30:00.000Z',
+          endDate: '2024-10-14T12:00:00.000Z',
+          offset: 0,
```

Comment:
❓ Could you perhaps explain these changes? It seems like the times are now off by two hours here compared to before (before: 10:30 in local time/UTC + 2, now: 10:30 in UTC). Or was the offset taken into account before and therefore basically also made it 10:30 in UTC?